src/test/java/org/sagebionetworks/bridge/reporter/worker/RetentionReportGeneratorTest.java

```diff
@@ -51,7 +56,7 @@
     RetentionReportGenerator generator;
     
     @Mock
-    private BridgeHelper bridgeHelper;
+    private static BridgeHelper bridgeHelper;
```

Comment:
I don't think we want to make this static. I'm not sure how Mockito handles this, and I think it would be safer to make it non-static, to ensure it's not re-used across test cases.